PluginBuilder/Program.cs

```diff
@@ -115,6 +113,7 @@ public void AddServices(IConfiguration configuration, IServiceCollection service
             options.Lockout.AllowedForNewUsers = true;
             options.Password.RequireUppercase = false;
         })
+        .AddTokenProvider("Default", typeof(SimpleTwoFactorTokenProvider<IdentityUser>))
```

Comment:
The token is easy to fake as anybody can know the token without seeing the mail.

I'm not sure this is a good idea. Probably better to do as we do in BTCPay.

Reply:
Nah, you would need to know the hash and this will be only for admin-initiated password reset... but I get what you are saying, I'm also using plain SHA256 and fixed variable that is visible in source code. I'll simplify it and do it in a less retarded way.

Reply:
c405efe